src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -372,7 +372,7 @@
     <value>Scroll to the bottom of history</value>
   </data>
   <data name="ScrollToPreviousMarkCommandKey" xml:space="preserve">
-    <value>Scroll to the the previous mark</value>
+    <value>Scroll to the previous mark</value>
   </data>

```

Comment:
oof. Yeah uh, glad the spell bot caught this one haha

Reply:
ha! This will be asking for a backport to 1.15